app/models/GoodsRecordsPagination.scala

```diff
@@ -32,47 +31,67 @@ case class GoodsRecordsPagination(
 
 object GoodsRecordsPagination {
   implicit val format: OFormat[GoodsRecordsPagination] = Json.format[GoodsRecordsPagination]
+  private val defaultRecord                            = 0
 
-  private def getPageSize(totalRecords: Int, recordsSize: Int, totalPages: Int): Int =
-    (totalRecords - recordsSize) / (totalPages - 1)
+  def getFirstRecord(pagination: GoodsRecordsPagination, pageSize: Int): Int =
```

Comment:
Maybe a slightly different name that makes it clear its like a goods record index and not the record itself? Seeing the calls for this I was expecting it to return a record, not an int

Reply:
Fixed